### PR TITLE
Show_demo

### PR DESCRIPTION
**Pull request by elandini84**
Added 3 xml for demo purposes. Each xml (except for `base_demo.xml` which is used by all the other ones) is related to a specific CER model.

### CODE REVIEW

File: app/show_demo/conf/look_around_pid.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+15800
```
Comment (pattacini) [inline app/show_demo/conf/look_around_pid.txt]: A file w/ a single number?

File: app/show_demo/conf/swing_arms_pid.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+12274
```
Comment (pattacini) [inline app/show_demo/conf/swing_arms_pid.txt]: Same comment as above.

**Review by pattacini — APPROVED** [review]
Just a few spot comments.

As a general remark, we tend to use funny-things to store such specific contents that are very demo-oriented.

I think we could merge the PR anyway.
I would only avoid adding further demo stuff in the future.

**Comment by elandini84** [comment]
> Just a few spot comments.
> 
> As a general remark, we tend to use funny-things to store such specific contents that are very demo-oriented.
> 
> I think we could merge the PR anyway.
> I would only avoid adding further demo stuff in the future.

Absolutely. I wasn't aware of this fact. If anyone thinks that the PR should be closed and put somewhere else it's completely fine with me. Sorry for the mistake.

File: app/show_demo/conf/swing_arms_pid.txt (hunk 2 above)
Comment (elandini84) [reply app/show_demo/conf/swing_arms_pid.txt]: Ok, that's an error. These are just files created by the bash scripts. I'll remove them immediately.

**Comment by pattacini** [comment]
>Absolutely. I wasn't aware of this fact. If anyone thinks that the PR should be closed and put somewhere else it's completely fine with me. Sorry for the mistake.

Never mind!
If you think you can get away with less, yeah that would be preferable, and very much appreciated @elandini84 👍🏻

**Comment by pattacini** [comment]
Hi @elandini84 

Any chance you could retarget this PR onto `funny-things`?

**Comment by elandini84** [comment]
> Hi @elandini84
> 
> Any chance you could retarget this PR onto `funny-things`?

I needs to be slightly modified in order to be retarget onto `funny-things`. I'll proceed as soon as possible. Given that, I think we can close this PR, can't we?

**Comment by pattacini** [comment]
>Given that, I think we can close this PR, can't we?

Yup, thanks heaps!

**Pull request closed without merging**